usr.sbin/bhyveload/bhyveload.c

```diff
@@ -160,15 +160,22 @@ cb_open(void *arg __unused, const char *filename, void **hp)
 {
 	struct cb_file *cf;
 	char path[PATH_MAX];
+	size_t nlen;
 
 	if (!host_base)
 		return (ENOENT);
 
 	strlcpy(path, host_base, PATH_MAX);
-	if (path[strlen(path) - 1] == '/')
-		path[strlen(path) - 1] = 0;
+
+	nlen = strlen(path) - 1;
```

Comment:
I'd prefer this be more like: len = strlen(path); if (path[len - 1] == '/') path[len-1]='\0';

Reply:
Thanks! I've updated it.